quattor/types/aquilon/system.pan

```diff
@@ -127,9 +129,9 @@ type structure_personality = {
     # want anything else.
     "maintenance_threshold" ? long(0..100) = 50
     "backups"       ? string
-    "host_environment" ? string with match(SELF, "^(dev|qa|uat|prod|infra|legacy)$")
-    "owner_eon_id" ? long
-    "stage"         ? string
+    "host_environment" : string with match(SELF, "^(dev|qa|uat|prod|infra|legacy)$")
+    "owner_eon_id"  : long
+    "stage"         : string
```

Comment:
Making some of the mandatory could cause issues for users on an older version of the broker

Reply:
See https://github.com/quattor/template-library-core/pull/177#discussion_r192972142

Reply:
OK! In that case then we should also remove infra and legacy options are they are no longer valid.